home_work_17/index.js

```diff
@@ -0,0 +1,169 @@
+const selectElement = document.querySelector("#select-field");
+const btnGetWeather = document.querySelector(".get-weather");
+const cityInfoContainer = document.querySelector(".city-info");
+const weatherContainer = document.querySelector(".weather-container");
+const neighborsContainer = document.querySelector(".neighbors-container");
+const neighborsCity = document.querySelector(".neighbors-city");
+
+const img = {
+  sunny: "https://developer.accuweather.com/sites/default/files/01-s.png",
+  cloudy: "https://developer.accuweather.com/sites/default/files/07-s.png",
+  partlySunny: "https://developer.accuweather.com/sites/default/files/03-s.png",
+  rain: "https://developer.accuweather.com/sites/default/files/18-s.png",
+};
```

Comment:
Optimise it:

`const imgURL = 'https://developer.accuweather.com/sites/default/files/'`

```suggestion
const img = {
  sunny: `${imgURL}01-s.png`,
  cloudy: "`${imgURL}07-s.png`,
  partlySunny: `${imgURL}03-s.png`,
  rain: `${imgURL}18-s.png`,
};
```